src/Elmish.WPF.Tests/BindingTests.fs

```diff
@@ -3763,3 +3763,69 @@ module subModelSelectedItem =
         let d = Binding.subModelSelectedItem("", (fail: _ -> _ option), set) |> getSubModelSelectedItemData
         test <@ d.Set (p |> Option.map box |> ValueOption.ofOption) m = set p @>
       }
+
+
+
+module sorting =
+
+  [<Fact>]
+    let ``SubModelSelectedItemData sorted last`` () =
+      //Property.check <| property {
+      //  let! oneWayData = GenX.auto<OneWayData<obj, obj>>
+      //  let! oneWayLazyData = GenX.auto<OneWayLazyData<obj, obj, obj>>
+      //  let! oneWaySeqLazyData = GenX.auto<OneWaySeqLazyData<obj, obj, obj, obj>>
+      //  let! twoWayData = GenX.auto<TwoWayData<obj, obj, obj>>
+      //  let! twoWayValidateData = GenX.auto<TwoWayValidateData<obj, obj, obj>>
+      //  let! cmdData = GenX.auto<CmdData<obj, obj>>
+      //  let! cmdParamData = GenX.auto<CmdParamData<obj, obj>>
+      //  let! subModelData = GenX.auto<SubModelData<obj, obj, obj, obj>>
+      //  let! subModelWinData = GenX.auto<SubModelWinData<obj, obj, obj, obj>>
+      //  let! subModelSeqData = GenX.auto<SubModelSeqData<obj, obj, obj, obj, obj>>
+      //  let! subModelSelectedItemData = GenX.auto<SubModelSelectedItemData<obj, obj, obj>>
+      //  let data =
+      //    [ SubModelSelectedItemData subModelSelectedItemData
+      //      OneWayData oneWayData
+      //      OneWayLazyData oneWayLazyData
+      //      OneWaySeqLazyData oneWaySeqLazyData
+      //      TwoWayData twoWayData
+      //      TwoWayValidateData twoWayValidateData
+      //      CmdData cmdData
+      //      CmdParamData cmdParamData
+      //      SubModelData subModelData
+      //      SubModelWinData subModelWinData
+      //      SubModelSeqData subModelSeqData
+      //      SubModelSelectedItemData subModelSelectedItemData
+      //    ]
+      //  let sorted = data |> List.sortWith BindingData.subModelSelectedItemDataLast
+      //  match sorted with
+      //  | [_; _; _; _; _; _; _; _; _; _; SubModelSelectedItemData _; SubModelSelectedItemData _] -> ()
+      //  | _ -> failwith "SubModelSelectedItemData was not sorted last"
+      //}
```

Comment:
Just leaving an unresolved conversation here as a reminder.

Reply:
I removed the commented out code, used `fail` and `fail2` in place of the functions I had, and used `GenX.atuo` to create `bool`, `string`, and `obj voption` instances.